var/spack/repos/builtin/packages/h5hut/package.py

```diff
@@ -6,24 +6,33 @@
 from spack.package import *
 
 
-class H5hut(AutotoolsPackage):
+class H5hut(AutotoolsPackage, CMakePackage):
     """H5hut (HDF5 Utility Toolkit).
     High-Performance I/O Library for Particle-based Simulations."""
 
     homepage = "https://amas.psi.ch/H5hut/"
-    url = "https://amas.web.psi.ch/Downloads/H5hut/H5hut-2.0.0rc3.tar.gz"
+    url = "https://amas.web.psi.ch/Downloads/H5hut/H5hut-2.0.0rc7.tar.gz"
     git = "https://gitlab.psi.ch/H5hut/src.git"
+    maintainers("biddisco")
 
-    version("2.0.0rc3", sha256="1ca9a9478a99e1811ecbca3c02cc49258050d339ffb1a170006eab4ab2a01790")
-
+    version("2.0.0rc7", sha256="bc058c4817c356b7b7acfe386c586923103b90bdfa83575db3a91754767e6fab")
     version("master", branch="master")
 
     depends_on("c", type="build")  # generated
     depends_on("cxx", type="build")  # generated
     depends_on("fortran", type="build")  # generated
 
+    build_system(
+        conditional("cmake", when="@2:"),
+        "autotools",
```

Comment:
_If_ the only listed version is 2.0.0rc7, does it make sense to leave the autotools support?